lib/components/Checkbox.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import InputLabel from './InputLabel';
+// import './Checkbox.css';
+
+/**
+ * Checkbox with label
+ * 
+ * @param {string, required} id input's id and label's for attributes
+ * @param {string, required} label Text for the label
+ * @param {string} name defaults to 'checkboxs'. Sets name attribute
+ * @param {string} value Sets value attribute. If not provided the id is used.
+ * @param {bool} checked  If true, item is initially set to checked
+ * @param {bool} disabled If item, item is disabled
+ */
+export default function Checkbox({id, label, name, value, checked, disabled }) {
+  return (
+    <div>
+      <input
+        id={id}
```

Comment:
No I'm not 100% sure.  From a technical perspective, I'm 99% sure that they're functionally equivalent, but from a 508 perspective, I'm not sure.  We should definitely ask Galina.